front/src/common-app/components/footer/footer.styles.ts

```diff
@@ -8,8 +8,8 @@ export const footerContainer = css`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  height: ${spacing(37.5)};
-  padding: ${spacing(2.5)} ${spacing(12.5)};
+  height: ${spacing(20.5)};
+  padding: ${spacing(2)} ${spacing(12.5)};
```

Comment:
I think is not a good idea you use these properties in the container and each container element, don't necessary and are confusing. This is my propsed CSS for your analysis and study. :-)
`import { css } from 'emotion';
import { theme } from 'core/theme';

const { palette, spacing, breakpoints } = theme;
const color = palette.customPalette;

export const footerContainer = css`
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  background-color: ${color.secondary};
`;

// you have change the name in footer.component.tsx
export const logoContainer = css`
  display: flex;
  justify-content: center;
  align-items: center;
`;

export const bottomContainer = css`
  display: flex;
  justify-content: space-between;
  padding: ${spacing(3)};
  border-top: 1px solid ${color.primary};
  @media (max-width: ${breakpoints.values.md}px) {
    flex-direction: column;
    justify-content: center;
    align-items: center;
  }
`;

export const lemoncodeIcon = css`
  max-width: 100px;
  padding: ${spacing(3)} 0;
`;

export const iconContainer = css`
  display: flex;
  justify-content: space-between;
  width: 50%;
  margin: 0;
  padding: 0 0 ${spacing(1)};
  list-style: none;
  @media (min-width: ${breakpoints.values.sm}px) {
    width: 20%;
  }
  @media (min-width: ${breakpoints.values.lg}px) {
    width: 10%;
  }
`;

export const icon = css`
  font-size: 1.5rem;
  color: ${color.greyLight};
  &:hover {
    cursor: pointer;
    color: ${color.primary};
  }
`;

export const githubIcon = css`
  height: ${spacing(2.75)};
  fill: ${color.greyLight};
  &:hover {
    cursor: pointer;
    fill: ${color.primary};
  }
`;

export const copyright = css`
  color: ${color.greyLight};
  @media (max-width: ${breakpoints.values.xs}px) {
    margin-top: ${spacing(0.8)};
  }
`;
`